archival-python-version/install-helix.sh

```
@@ -0,0 +1,695 @@
+            echo "Installation aborted."
+            exit 1
+            ;;
+    esac
+}
+
+# Ask for user approval before proceeding
+ask_for_approval
+
+sudo mkdir -p $INSTALL_DIR
+# Change the owner of the installation directory to the current user
+sudo chown -R $(id -un):$(id -gn) $INSTALL_DIR
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)